TestRunner/Program.cs

```diff
@@ -0,0 +1,107 @@
+﻿using BH.oM.Reflection.Debugging;
+using BH.oM.Test.Results;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestRunner
+{
+    class Program
+    {
+        /*************************************/
+        /**** Main                        ****/
+        /*************************************/
+
+        static void Main(string[] args)
+        {
+            LoadAllTestAssemblies();
+
+            if (args.Length == 0)
+            {
+                Console.WriteLine("Please provide a filter for the methods yo uwant to run. This can be either the name of the method or its namespace (after 'BH.Test')");
```

Comment:
```suggestion
                Console.WriteLine("Please provide a filter for the methods you want to run. This can be either the name of the method or its namespace (after 'BH.Test')");
```